astroid/brain/brain_builtin_inference.py

```diff
@@ -892,6 +893,21 @@ def _build_dict_with_elements(elements):
     return _build_dict_with_elements([])
 
 
+def _infer_copy_method(
+    node: nodes.Call, context: Optional[InferenceContext] = None
+) -> Iterator[Union[nodes.Dict, nodes.List, nodes.Set, objects.FrozenSet]]:
+    inferred_orig, inferred_copy = itertools.tee(node.func.expr.infer(context=context))
```

Comment:
```suggestion
    assert isinstance(node.func, nodes.Attribute)
    inferred_orig, inferred_copy = itertools.tee(node.func.expr.infer(context=context))
```
This is already checked in `register_transform` but mypy doesn't know about it.